doc/source/index.rst

```diff
@@ -23,3 +23,16 @@ Index
     archiver
     whats_new
     DBEP/index
+
+.. toctree::
+   :hidden:
+   :caption: Bluesky Project
+
+   Bluesky Project <https://blueskyproject.io>
+   Bluesky Project Repositories <https://github.com/bluesky>
+
+.. toctree::
+   :hidden:
+   :caption: Getting Help
+
+   Gitter <https://gitter.im/NSLS-II/DAMA>
```

Comment:
Actually --- In the spirit of scrubbing facility-specific branding out, should we move to a bluesky gitter? The way projects do this (e.g. jupyterlab moved from jupyter/jupyterlab -> jupyterlab/jupyterlab) is to post a big message in the gitter with a link to the new one. There is no way to set up an actual redirect.

Reply:
If this were _mainly_ an NSLS-II support channel I would say we should leave it as is, but it seems that a fair fraction of the questions we get are from other facilities that are evaluating bluesky.

Reply:
I agree